internal/storage/index/builder.go

```diff
@@ -136,6 +136,8 @@ func Build(ctx context.Context, fsys fs.FS, opts ...BuildOpt) (Index, error) {
 type indexBuilder struct {
 	executables  map[namer.ModuleID]struct{}
 	modIDToFile  map[namer.ModuleID]string
+	modIDTOFQN   map[namer.ModuleID]string
+	FQNToModID   map[string]namer.ModuleID
```

Comment:
This is unnecessary because it's easy to calculate the module ID given the FQN. Let's not use up memory to store something that can be easily calculated.